dev-libs/libmanette/libmanette-0.2.12.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome.org meson vala xdg
+
+DESCRIPTION="Simple GObject game controller library"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/libmanette"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="doc +introspection +udev +vala test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="vala? ( introspection ) doc? ( introspection )"
+
+RDEPEND="
+	>=dev-libs/glib-2.50:2
+	udev? ( dev-libs/libgudev[introspection?] )
+	dev-libs/libevdev
+	dev-libs/hidapi
+	introspection? ( >=dev-libs/gobject-introspection-1.83.2:= )
+"
+DEPEND="${RDEPEND}"
+# See https://bugs.gentoo.org/952906 for >=dev-lang/vala-0.56.18
```

Comment:
Not sure this makes sense? That's not something we want to add to every single package that uses Vala (and if it is, that's something to be done separately).

Reply:
Thanks!